Storage/Processor/Processor.php

```diff
@@ -219,14 +219,19 @@ public function getResource(Config $config, string $filename)
             return $this->storageManager->getResource($config->getAssetHash(), $config->getStorageContext());
```

Comment:
Should this not be try catched?

What about defining the 2nd parameter of getResource first, and then calling the function? (merging this line and line 225)

Reply:
I added a try catch with a warning log